cudapoa/include/claraparabricks/genomeworks/cudapoa/utils.hpp

```diff
@@ -143,31 +143,16 @@ inline void parse_fasta_windows(std::vector<std::vector<std::string>>& windows,
 {
```

Comment:
nit-picky: to keep the parser function names consistent, what do you think of renaming `parse_window_data_file()` and `parse_fasta_windows()` to something like `parse_cudapoa_file()` and `parse_fasta_files()`?